src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsAbcdProductsPriority.java

```diff
@@ -52,8 +53,8 @@ public void adsUAPFamilyAdIsNotDisplayedOnFeaturedVideoPage() {
   }
 
   private void verifyNoUAPSizesInSlot(AdsBaseObject ads, String slot) {
-    Assert.assertFalse(String.format("Slot %s has UAP supported 2x2 size", slot), ads.slotHasSize(slot, 2, 2));
-    Assert.assertFalse(String.format("Slot %s has UAP supported 3x3 size", slot), ads.slotHasSize(slot, 3, 3));
+    Assert.assertFalse(ads.slotHasSize(slot, 2, 2), String.format("Slot %s has UAP supported 2x2 size", slot));
+    Assert.assertFalse(ads.slotHasSize(slot, 3, 3), String.format("Slot %s has UAP supported 3x3 size", slot));
```

Comment:
I wouldn't expect this changes here. Was it intended?

Reply:
it was, since the testng upgrade

Reply:
Cool. Better safe than sorry